### PR TITLE
Updated the "About" section

### PR DESCRIPTION
**Pull request by Prime-5**
I'm a little new to OpenSource and am learning to contribute in the only way I know right now. I hope you guys find it helpful.

### CODE REVIEW

**Comment by alanna** [comment]
Thank you for the suggested changes. I will leave it to @contraexemplo to review as they are leading the refresh of our docs.

File: about/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-Meetups, open source projects, parent associations, neighborhood associations, pet projects, clubs, unions, movements, non-profits, business incubators - in order to operate, all of them are forced to use a physical glass jar, ask a sponsor to directly pay their expenses or front the huge overhead of setting up and managing a corporation or a non-profit. It’s either inefficient and opaque or it’s overkill.
+To be able to operate, meetups, open-source projects, parent associations, neighborhood associations, pet projects, clubs, unions, movements, non-profits, business incubators - all are either forced to use a physical glass jar, asking a sponsor to directly pay for their expenses, or front the huge overhead of setting up and managing a corporation or a non-profit. Even if they manage to achieve it in a way that in neither inefficient nor opaque, it is overkill.
```
Comment (contraexemplo) [inline about/README.md]: Small typo: 
Shouldn't it be "Even if they manage to achieve it in a way that **is** neither inefficient nor opaque, it is overkill."?

File: about/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-Open Collective enables groups to quickly set up a collective, raise funds and manage them transparently.
+This is the problem which Open Collective solves. We enable groups to quickly set up a collective, raise funds and manage them transparently.
```
Comment (contraexemplo) [inline about/README.md:7]: This a nice revamp.

File: about/README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-We want all those seeds to have a chance to grow, to bring their ideas to life. We believe everyone should have the tools to create the organizations of tomorrow. And we are dedicating ourselves to making that happen!
+We want all those seeds to have a chance to grow and give the ideas a fighting chance to reach their true potential. We believe everyone should have the tools to create the organizations of tomorrow. And we are dedicating ourselves to make that happen!
```
Comment (contraexemplo) [inline about/README.md:9]: Those are also some really nice additions. Well done!

**Review by contraexemplo — CHANGES_REQUESTED** [review]
Hello @Prime-5, thank you so much for making those changes! I believe there's a small typo in the second paragraph you edited, could you take a look into that? Apart from that, I liked your revamp a lot.

File: about/README.md (hunk 2 above)
Comment (contraexemplo) [reply about/README.md:7]: Great, thank you. :)

**Review by contraexemplo — APPROVED** [review]

**Comment by Prime-5** [comment]
Hello @contraexemplo, thank you for the feedback! Actually, this is my first pull request on GitHub and I'm glad it turned out to be useful. 
Also, I know GSoD is going on and there must be plenty of stuff on your plate, but is it okay if I tweak some documentation once in a while? (Just to get a hang of how technical documentation works).

**Comment by contraexemplo** [comment]
@Prime-5 Nice! Happy first merge! :tada:

> is it okay if I tweak some documentation once in a while?

I encourage you to do so, actually! At the moment I'm focusing my efforts on https://docs.opencollective.com (and documenting that work [on my own GitBook](https://metadocumentation.gitbook.io/metadocumentation/)). Having other people submitting feedback and suggesting changes is greatly appreciated.

**Comment by Prime-5** [comment]
@contraexemplo Thank you :)

**Comment by jaskiratsingh2000** [comment]
Great work.

**Pull request merged**